app/models/account.rb

```diff
@@ -21,7 +21,6 @@ class Account < ActiveRecord::Base
   validates :url, uniqueness: true
 
   after_create :set_metadata_annotation, :create_source, :set_provider
-  after_commit :update_elasticsearch_account, on: :update
```

Comment:
Not sure why we would emove accounts from the ES schema. What's the idea here?

Reply:
The parent of these accounts in ES is `ProjectSource` type so I removed it as the parent will not created but the account related to media still created and indexed in ES.

Reply:
sounds good thanks